app/javascript/modules/listingDetails/ListingDetailsImageCard.tsx

```diff
@@ -50,17 +50,20 @@ export const ListingDetailsImageCard = ({ listing }: ListingDetailsImageCardProp
               </a>
             </p>
           </div>
-          <a
-            href={`${getShareListingPath()}/${listing.listingID}`}
-            onBlur={() => setShareImage(shareButton)}
-            onFocus={() => setShareImage(shareButtonSelected)}
-            onMouseEnter={() => setShareImage(shareButtonSelected)}
-            onMouseLeave={() => setShareImage(shareButton)}
-            target="_blank"
-            className="share-button"
-          >
-            <img alt={t("label.share")} src={shareImage} />
-          </a>
+          <div>
+            <a
```

Comment:
I might just being blind! But I can't find this in the Review Instructions.

Reply:
Ugh, I borked a rebase. We need to add aria labels to the link and alt text to the image, but adding this div should not be a part of it. Thank you for catching!

Reply:
ha no worries! I totally understand